src/test/java/com/mendix/audittrail/tests/TestAuditCompany.java

```diff
@@ -0,0 +1,53 @@
+package com.mendix.audittrail.tests;
+
+import audittrail.proxies.MemberType;
+import audittrail.proxies.TypeOfLog;
+import com.mendix.audittrail.tests.expected.ExpectedLog;
+import com.mendix.core.CoreException;
+import com.mendix.systemwideinterfaces.core.IMendixObject;
+import test_crm.proxies.Company;
+import test_crm.proxies.Group;
+
+import java.util.Arrays;
+
+import static test_crm.proxies.Company.MemberNames.*;
+import static test_crm.proxies.Company.MemberNames.Company_Group;
+
+/**
+ * Helper class for creating tests with a company objects
+ */
+public abstract class TestAuditCompany extends TestAuditBase {

```

Comment:
I'd name this `TestAuditWithData` so it better reflects the purpose of the class